sdk/containerregistry/azure-containers-containerregistry/src/main/java/module-info.java

```diff
@@ -2,5 +2,6 @@
 // Licensed under the MIT License.
 
 module com.azure.containers.containerregistry {
+    requires transitive com.azure.core;
     exports com.azure.containers.containerregistry;
```

Comment:
If you want the classes in `com.azure.containers.containerregistry.authentication` to be available outside of the project you'll need to add it as an export here.

Reply:
Don't want that right now.

---
In reply to: [598005797](https://github.com/Azure/azure-sdk-for-java/pull/19926#discussion_r598005797) [](ancestors = 598005797)